afdko/.SRCINFO

```diff
@@ -1,21 +1,28 @@
 pkgbase = afdko
 	pkgdesc = Adobe Font Development Kit for OpenType
 	pkgver = 4.0.1
-	pkgrel = 1
+	pkgrel = 6
```

Comment:
Please don't keep bumping the pkgrel. Obviously you many need to bump it locally once to test, and I think we can also bump it *once* in this PR, but I'm not going to skip from 1 to 6 and the only releases that count are the ones in the AUR ... until this merges that's currently at 1.